stacks/cloud-manager/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
Please make this file an executable. `chmod +x`

Reply:
Fixed this as a second PR (3-2). (Ignore PR3...no need to review)
Please let me know if this fixes work for you, and you manage to deploy the app.
Thanks,
J.